testing/tpm2-tools/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor:
+# Maintainer: Alexander Sack <asac@pantacor.com>
+pkgname="tpm2-tools"
+pkgver="4.0_rc2"
+pkgrel=0
+pkgdesc="TPM2 Tools"
+url="https://github.com/tpm2-software/tpm2-tools/"
+arch="all"
+license="BSD-2-Clause"
+makedepends="tpm2-tss-dev automake autoconf autoconf-archive libtool openssl-dev doxygen linux-headers curl-dev"
+subpackages="tpm2-tools-doc"
+sourcever="4.0-rc2"
```

Comment:
Wouldn't ${pkgver/_/-} be easier to keep up-to-date?

Reply:
thanks for the trick.